app/forms/webauthn_setup_form.rb

```diff
@@ -64,11 +65,11 @@ def consume_parameters(params)
   def name_is_unique
     return unless WebauthnConfiguration.exists?(user_id: @user.id, name: @name)
     if @platform_authenticator
-      errors.add :name, I18n.t('errors.webauthn_platform_setup.unique_name'), type: :unique_name
+      @name.concat('-').concat(Time.zone.now.strftime(' %d%m%Y'))
```

Comment:
- Can we add some test coverage for this behavior?
- The formatting of this is a little strange to me, e.g. I get something like `"Chrome 119 on macOS 10- 30112023"` if it wasn't unique
   - The spacing around the dash is inconsistent
   - The set of numbers isn't very obvious that it's `[day][month][year]`, since that's not a very common date format (in the U.S. at least), and there's no delimiters

Ultimately, I don't know that it matters a whole lot that a user can understand it, though personally I'd imagined just an appended incrementing number like "Chrome 119 on macOS 10 (2)". Could be worth checking in with the UX folks to see if they have any opinions.

Reply:
Added test for the name_is_unique.
Changed the append structure to add '(1)' to non-unique names.